hazelcast/src/main/java/com/hazelcast/internal/util/ServiceLoader.java

```diff
@@ -374,8 +374,8 @@ private boolean isDuplicate(Class<?> candidate) {
         private void onClassNotFoundException(String className, ClassLoader classLoader, ClassNotFoundException e) {
             if (className.startsWith("com.hazelcast")) {
                 LOGGER.fine("Failed to load " + className + " by " + classLoader
-                        + ". This indicates a classloading issue. It can happen in a runtime with "
-                        + "a complicated classloading model. (OSGi, Java EE, etc);");
+                        + ". This indicates a classloading issue. It can happen at runtime with "
```

Comment:
I think the original was referring to a specific runtime environment (OSGi, Java EE) and not to execution time; so I would say that "in a runtime" was actually correct.

Reply:
I know, but it's wrong to take one of the possible reasons and say as if it was the only reason.

Reply:
I misunderstood your comment, I agree, I'll fix.